src/classes/Lists.ts

```diff
@@ -19,4 +26,130 @@ export default class Lists extends OneBlinkAPI {
     options = options || {}
     super(options.accessKey, options.secretKey)
   }
+
+  /**
+   * #### Example
+   *
+   * ```javascript
+   * const searchParams = {
+   *   limit: 1,
+   *   offset: 0,
+   * }
+   * const { lists, meta } = await lists.searchLists(searchParams)
+   * ```
+   *
+   * @param searchParams Search options
+   */
+  async searchLists(searchParams: ListSearchOptions): Promise<
+    Omit<ListSearchResult, 'formElementDynamicOptionSets'> & {
+      lists: FormTypes.FormElementOptionSet[]
+    }
+  > {
+    const result = await super.searchRequest<ListSearchResult>(
+      basePath,
+      searchParams,
+    )
+    return {
+      meta: result.meta,
+      lists: result.formElementDynamicOptionSets,
+    }
+  }
+
+  /**
+   * #### Example
+   *
+   * ```javascript
+   * const data = {
+   *   name: 'my list',
+   *   organisationId: 'abc123',
+   *   environments: [
+   *     {
+   *       options: [
+   *          {
+   *            label: 'One'
+   *            value: '1'
+   *          },
+   *          {
+   *            label: 'Two'
+   *            value: '2'
+   *          }
+   *       ]
+   *       formsAppEnvironmentId: 1,
+   *     },
+   *   ],
+   *   type: 'STATIC',
+   * }
+   * const list = await lists.createList(data)
+   * // Use list here...
+   * ```
+   *
+   * @param data The data for the new list
+   */
+  async createList(
+    data: FormTypes.NewFormElementOptionSet,
+  ): Promise<FormTypes.NewFormElementOptionSet> {
+    return super.postRequest(basePath, data)
```

Comment:
Prob should do the type check thing here too.

Reply:
The Email class didn't have it! (not a valid reason, I will add)